src/utils.ts

```diff
@@ -694,3 +695,12 @@ export const injectHostIntoSession = (
     webSocketDebuggerUrl,
   };
 };
+
```

Comment:
This is the *magic* here as the page._client is some new TS wizardry that tries to prevent outside access. `page.call` fixes that.